AGENTS.md

```
@@ -34,6 +34,21 @@ where `YYYY-MM-DD` equals the `> Completed:` marker inside the archived
 archiving, run `pnpm archive:index` to refresh
 `openspec/changes/archive/README.md`.
 
+When a task is deferred to a follow-up GitHub issue, annotate the
+checkbox in `tasks.md` with a single canonical Markdown blockquote
+on the next line:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)